products/libllbuild/include/llbuild/buildsystem.h

```diff
@@ -99,6 +99,7 @@ typedef enum {
 typedef struct llb_buildsystem_command_extended_result_t_ {
   llb_buildsystem_command_result_t result;  /// Result of command execution
   int exit_status;                          /// The exit code
+  int32_t pid;                              /// The process identifier (-1 if process creation failed)
```

Comment:
Although it shouldn't be a problem, for safety I'd prefer to have this field added to the bottom of the struct after maxrss.